### PR TITLE
#39265 covers

### PR DESCRIPTION
**Pull request by pbearne**
This a working branch and pull request for the adding of @covers to all the PHP test
This is not complete at this time 
I have merged in all the SVN patches but I have to check that they are OK as I got errors in the merge

Trac ticket: #39265

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: tests/phpunit/tests/file.php [hunk 1]
```diff
@@ -16,6 +16,8 @@ function setUp() {
 	/**
 	 * @group plugins
 	 * @group themes
+	 *
+	 * @cover ::get_file_data
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:20]:
```suggestion
	 * @covers ::get_file_data
```

File: tests/phpunit/tests/file.php [hunk 2]
```diff
@@ -46,6 +48,8 @@ function test_get_file_data() {
...
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:52]:
```suggestion
	 * @covers ::get_file_data
```

File: tests/phpunit/tests/file.php [hunk 3]
```diff
@@ -88,6 +92,9 @@ function is_unique_writable_file( $path, $filename ) {
 		return $result;
 	}
 
+	/**
+	 * @cover ::wp_unique_filename
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:96]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 4]
```diff
@@ -97,6 +104,9 @@ function test_unique_filename_is_valid() {
 		unlink( $this->dir . DIRECTORY_SEPARATOR . $filename );
 	}
 
+	/**
+	 * @cover ::wp_unique_filename
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:108]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 5]
```diff
@@ -113,6 +123,9 @@ function test_unique_filename_is_unique() {
 		unlink( $this->dir . DIRECTORY_SEPARATOR . $filename2 );
 	}
 
+	/**
+	 * @cover ::wp_unique_filename
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:127]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 6]
```diff
@@ -125,6 +138,9 @@ function test_unique_filename_is_sanitized() {
 		unlink( $this->dir . DIRECTORY_SEPARATOR . $filename );
 	}
 
+	/**
+	 * @cover ::wp_unique_filename
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:142]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 7]
```diff
@@ -138,6 +154,9 @@ function test_unique_filename_with_slashes() {
...
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:158]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 8]
```diff
@@ -150,6 +169,9 @@ function test_unique_filename_multiple_ext() {
...
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:173]:
```suggestion
	 * @covers ::wp_unique_filename
```

File: tests/phpunit/tests/file.php [hunk 9]
```diff
@@ -163,6 +185,8 @@ function test_unique_filename_no_ext() {
 
 	/**
 	 * @dataProvider data_wp_tempnam_filenames
+	 *
+	 * @cover ::wp_tempnam
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:189]:
```suggestion
	 * @covers ::wp_tempnam
```

File: tests/phpunit/tests/file.php [hunk 10]
```diff
@@ -183,6 +207,8 @@ function data_wp_tempnam_filenames() {
 
 	/**
 	 * @ticket 47186
+	 *
+	 * @cover ::verify_file_signature
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:211]:
```suggestion
	 * @covers ::verify_file_signature
```

File: tests/phpunit/tests/file.php [hunk 11]
```diff
@@ -216,6 +242,8 @@ function test_file_signature_functions_as_expected() {
...
```
Comment (jrfnl) [inline tests/phpunit/tests/file.php:246]:
```suggestion
	 * @covers ::verify_file_signature
```

File: tests/phpunit/tests/filters.php [hunk 12]
```diff
@@ -360,13 +414,18 @@ public function test_current_priority() {
 		$this->assertSame( 99, $this->current_priority );
 	}
 
+	/**
+	 * One tag with multiple filters.
```
Comment (jrfnl) [inline tests/phpunit/tests/filters.php:418]: Does this docblock belong here ?

File: tests/phpunit/tests/functions.php [hunk 13]
```diff
@@ -4,6 +4,11 @@
  * @group functions.php
  */
 class Tests_Functions extends WP_UnitTestCase {
+
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/functions.php:9]: Stray blank comment line should be removed.

File: tests/phpunit/tests/meta.php [hunk 14]
```diff
@@ -17,6 +17,10 @@ function _meta_sanitize_cb( $meta_value, $meta_key, $meta_type ) {
 		return 'sanitized';
 	}
 
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/meta.php:21]: Stray blank comment line should be removed.

File: tests/phpunit/tests/meta.php [hunk 15]
```diff
@@ -26,6 +30,10 @@ function test_sanitize_meta() {
 		$this->assertSame( 'sanitized', $meta );
 	}
 
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/meta.php:34]: Stray blank comment line should be removed.

File: tests/phpunit/tests/meta.php [hunk 16]
```diff
@@ -41,6 +49,10 @@ function test_delete_metadata_by_mid() {
 		$this->assertFalse( (bool) get_user_meta( $this->author->ID, 'delete_meta_key' ) );
 	}
 
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/meta.php:53]: Stray blank comment line should be removed.

File: tests/phpunit/tests/meta.php [hunk 17]
```diff
@@ -104,6 +118,10 @@ public function test_update_metadata_hooks_for_multiple_updated_rows() {
 		}
 	}
 
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/meta.php:122]: Stray blank comment line should be removed.

File: tests/phpunit/tests/meta.php [hunk 18]
```diff
@@ -198,6 +220,10 @@ function test_user_metadata_not_exists() {
 		);
 	}
 
+	/**
+	 *
```
Comment (jrfnl) [inline tests/phpunit/tests/meta.php:224]: Stray blank comment line should be removed.

**Review by jrfnl — COMMENTED** [review]
Hi @pbearne Thanks for setting up this PR. I've turned the build on so you can see the results of the test run and done a cursory look through of the PR and left some comments.

**Comment by pbearne** [comment]
This works as a better way to track the updates

File: tests/phpunit/tests/filters.php (hunk 12 above)
Comment (pbearne) [reply tests/phpunit/tests/filters.php:418]: yes it as stay merge removed

**Pull request closed without merging**